new_dawn_server/users/api/tests/test_beams_notification.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from django.test import TestCase
+from new_dawn_server.users.models import User
+from tastypie.test import ResourceTestCaseMixin
+
+
+# MultiAuthentication works when any of its authentication method is fulfilled
+class NotificationTest(ResourceTestCaseMixin, TestCase):
+    def setUp(self):
+        super().setUp()
+        self.register_arguments = {
+            "first_name": "test",
+            "last_name": "user",
+            "username": "test-user",
+            "password": "test-pwd",
+        }
+        res = self.api_client.post(
+            "/api/v1/register/", format="json", data=self.register_arguments)
+        res_body = json.loads(res.content)
+        self.user_id = res_body["id"]
+
+    def test_authenticate(self):
+        self.assertEqual(User.objects.count(), 1)
+        # Positive: authentication successful
+        res = self.api_client.get("/api/v1/user/notification/authenticate/?user_id=1")
+        res_body = json.loads(res.content)
+        self.assertIn("token", res_body)
```

Comment:
Will this push a real notification? If so please patch the method.

Reply:
It will only send back a token: 
{
    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovLzQ2NmNiNWRlLTBmZDItNDBjNy1hZDQ1LTgwMmI2Yzc5NTUwZS5wdXNobm90aWZpY2F0aW9ucy5wdXNoZXIuY29tIiwic3ViIjoiMSIsImV4cCI6MTU1NDE1MDYxMX0.LlvoVvRx7DaL2ZE5KVL-350lZmv4KPJ70FL7eRhvTRY"
}

Reply:
I’m not sure if we’ll be charged for calling this a lot of times. If not then we can leave it as is

Reply:
This is beams plan. We can send as many as we want.
<img width="1136" alt="Screen Shot 2019-03-31 at 1 16 32 PM" src="https://user-images.githubusercontent.com/40241750/55292319-3849ec00-53b7-11e9-8a7c-50441169d10f.png">